client/src/components/Layout.js

```diff
@@ -54,7 +54,7 @@ export const Layout = ({ children }) => {
   const showWide = widePaths.includes(router.pathname)
 
   // exclude the padding / and box shadow on the following pages
-  const excludPadPaths = ['/', '/about']
+  const excludPadPaths = ['/', '/about', '/contribute']
```

Comment:
I think removing contribute here would make the page more consistent with the designs.
We typically only have the pad removed on pages where the hero should bump up against the nav bar.